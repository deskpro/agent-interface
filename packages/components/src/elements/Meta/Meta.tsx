import * as React from "react";

export type MetaProps = {
  name: string;
  value: string;
};

const Meta: React.FC<MetaProps> = ({ name, value }) => (
  <span className="dp-Meta">
    {name}
    {": "}
    <span className="dp-Meta-value">{value}</span>
  </span>
);

export default Meta;
