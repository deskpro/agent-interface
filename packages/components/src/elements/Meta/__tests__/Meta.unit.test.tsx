import * as React from "react";
import { render } from "react-testing-library";

import Meta from "../Meta";

describe("<Meta />", () => {
  it("it should match its snapshot", () => {
    const { container } = render(<Meta name="Name" value="Value" />);
    expect(container.querySelector(".dp-Meta")).toMatchSnapshot();
  });
});
