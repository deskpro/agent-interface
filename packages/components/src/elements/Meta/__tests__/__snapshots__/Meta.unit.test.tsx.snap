// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Meta /> should match its snapshot 1`] = `
<span
  class="dp-Meta"
>
  Name
  : 
  <span
    class="dp-Meta-value"
  >
    Value
  </span>
</span>
`;
